### PR TITLE
explain how to verify the image download

### PR DESCRIPTION
**Pull request by kkaempf**
Signed-off-by: Klaus Kämpf <kkaempf@suse.de>

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -286,7 +286,21 @@ process is modelled similar to x86-64. You boot from a seed image (USB stick in
 wget -q https://download.opensuse.org/repositories/isv:/Rancher:/Elemental:/Dev:/Teal53/images/rpi.raw
 ```
 
-This `.raw` image needs to be written to a USB stick to boot from. This can be done with `dd` on the Linux command line if you're comfortable with this command.
+##### Verifying the download
+
+There are reports of incomplete downloads. You should verify your
```
Comment (Itxaka) [inline docs/quickstart.md:291]: I would rephrase this as currently sounds too negative. How about?

"In order to verify the integrity of the downloaded artifacts" 

and remove the "there are reports of incomplete downloads" ?

Comment (kkaempf) [reply docs/quickstart.md]: 👍🏻
done

**Review by Itxaka — APPROVED** [review]
